packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -41,6 +41,10 @@ function getSecurityCodeText(language: string | undefined) {
     case 'es-LA': {
       return 'Código de seguridad de Celo'
     }
+    case 'pt':
```

Comment:
I chatted with the mobile circle and there isn't really a standardization for how local is represented across various services.  The language for the phone’s OS is in the format of 'es-US' 'es-LA' .  But for Valora's language settings, it’s actually es-419.  The Valora app could probably pass either to this endpoint, and I'd think that the Valora language settings makes the most sense.

I don't see anything wrong with supporting all cases for now and cleaning up later (in case we find one format to be preferable).

Additionally, we could do something like:

```
delimiterIndex = language.indexOf("-")
language.substring(0, delimiter >=0 ? delimiter : language.length - 1 )
```

so we reduce our cases to just the language 'es', 'pt', ect.

Reply:
I was thinking we may end up with different messages for e.g. LatAm vs Castilian spanish so we might prefer to just keep the flexible structure right now and make sure that whatever Valora sends matches one of these values.

Reply:
kk seems fine to keep as it for now 👍